docs/requirements.txt

```diff
@@ -2,3 +2,9 @@ docutils==0.17.1
 sphinx==5.3.0
 sphinx-rtd-theme==1.1.1
 sphinx_autodoc_typehints==1.19.5
+sphinx-design==0.4.1
+ipykernel==6.23.1
+matplotlib==3.71
```

Comment:
I'm on an old Ubuntu with Py3.9, so this might not be supported anyway but I get 
```
ERROR: Could not find a version that satisfies the requirement matplotlib==3.71
```

Reply:
Oh, should most definitely have been 3.7.1. Thanks for the discovery.